mscore/musescore.cpp

```diff
@@ -4488,13 +4488,20 @@ AboutBoxDialog::AboutBoxDialog()
       revisionLabel->setText(tr("Revision: %1").arg(revision));
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
+      QString visitAndDonateString;
+#if !defined(FOR_WINSTORE)
+      visitAndDonateString = tr("Visit %1www.musescore.org%2 for new versions and more information.\n"
+                                        "Support MuseScore with your %3donation%4.")
+                                     .arg("<a href=\"https://www.musescore.org/\">")
+                                     .arg("</a>")
+                                     .arg("<a href=\"https://www.musescore.org/donate\">")
+                                     .arg("</a>").replace("\n", "<br/>");
+      visitAndDonateString += "<br/><br/>";
+#endif
+      QString copyrightString = tr("Copyright &copy; 1999-2019 Werner Schweer and Others.\n"
+                                   "Published under the GNU General Public License.").replace("\n", "<br/>");
       copyrightLabel->setText(QString("<span style=\"font-size:10pt;\">%1</span>")
-                              .arg(tr("Visit %1www.musescore.org%2 for new versions and more information.\n").arg("<a href=\"http://www.musescore.org/\">")
-                                   .arg("</a>").replace("\n", "<br/>") +
-                                   tr("Support MuseScore with your %3donation%4.\n\n").arg("<a href=\"http://www.musescore.org/donate\">")
-                                   .arg("</a>").replace("\n", "<br/>") +
-                                   tr("Copyright &copy; 1999-2019 Werner Schweer and Others.\n").replace("\n", "<br/>") +
-                                   tr("Published under the GNU General Public License.").replace("\n", "<br/>")));
+                              .arg(visitAndDonateString + copyrightString));
```

Comment:
Does this work with `visitAndDonateString` being potentially empty and not initialized?

Reply:
Sure, `QString` is just empty if initialised with default constructor.

Reply:
Done